docs/supported-version.md

```diff
@@ -0,0 +1,15 @@
+# Xen Orchestra compatibility
+
+Xen Orchestra is designed to work exclusively on XCP-ng and Citrix XenServer hypervisor. Theorically, Xen Orchestra is fully functionnal with any version of these two hypervisor. However, to benefit from the best support quality, our product is tested to work the best with the following versions:
+
+## Citrix XenServer (Citrix hypervisor)
+
+* XenServer 7.4 (standard & enterprise edition)
```

Comment:
No need to specify the edition, we could say "All edition" in the start, once for all